tests/unit/test_bigquery_connection_manager.py

```diff
@@ -53,7 +53,7 @@ def generate_connection_reset_error():
         assert new_mock_client is not self.mock_client
 
     def test_is_retryable(self):
-        _is_retryable = dbt.adapters.bigquery.retry._is_retryable
+        _is_retryable = dbt.adapters.bigquery.retry._DeferredException(1)._is_retryable
```

Comment:
`_is_retryable` was moved from a function to a method on `_DeferredException`, otherwise this test should still work the same.